docs/ibc/light-clients/setup.md

```diff
@@ -0,0 +1,110 @@
+<!-- 
+order: 4
+-->
+
+# Setup
+
+Learn how to confgure light client modules and create clients using core IBC and the `02-client` submodule. {synopsis}
+
+## Configuring a light client module
+
+An IBC light client module must implement the [`AppModuleBasic`](https://github.com/cosmos/cosmos-sdk/blob/main/types/module/module.go#L50) interface in order to register its concrete types against the core IBC interfaces defined in `modules/core/exported`. This is accomplished via the `RegisterInterfaces` method which provides the light client module with the opportunity to register codec types using the chain's `InterfaceRegistery`. Please refer to the [`07-tendermint` codec registration](https://github.com/cosmos/ibc-go/blob/7ae97694af558db2924e66ce4ce6527c03da89a5/modules/light-clients/07-tendermint/codec.go#L11).
+
+The `AppModuleBasic` interface may also be leveraged to install custom CLI handlers for light client module users.
+
+<!-- TODO: link to how to configure in app.go -->
```

Comment:
currently the doc should be able to be accessed via `../itegration.md#integrating-light-clients`. once #2905 is merged.

Maybe we can link there, and add a section here that simply says to register the light client module alongside or in place of `ibctm.AppModuleBasic{}`

Reply:
updated! :)